lma-integration/tests/Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM ubuntu:groovy
+FROM ubuntu:impish
```

Comment:
I fear that we're going to forget to update this. Do you think we could use `ubuntu:devel` here?

Reply:
Ah, good point.  I've adjusted the Dockerfile.